client/layout/masterbar/woo-core-profiler.tsx

```diff
@@ -62,12 +63,12 @@ const WooCoreProfilerMasterbar = ( { translate }: { translate: ( text: string )
 							</a>
 						</li>
 						<li className="masterbar__woo-nav-item">
-							{ shouldShowNoThanks && typeof redirectTo === 'string' && redirectTo.length && (
+							{ shouldShowNoThanks && typeof redirectTo === 'string' && isWebUri( redirectTo ) && (
```

Comment:
I don't think we need `typeof redirectTo === 'string'`, because we don't access `.length` anymore.

```suggestion
							{ shouldShowNoThanks &&  isWebUri( redirectTo ) && (
```

Reply:
Actually, We need it because `redirectTo` could be `null`.  🙂

Reply:
That's OK, if types are complaining it means we added incorrect types. 

```ts
declare module 'valid-url' {
	export function isWebUrl( url: string | null ): boolean;
}
```

The module [accepts all falsy values](https://github.com/ogt/valid-url/blob/master/index.js#L23).

Reply:
Good catch! I think we still need `typeof` for `href`.

Reply:
That makes sense! Let's ship 🚢